GG/src/dialogs/ColorDlg.cpp

```diff
@@ -614,34 +614,38 @@ void ColorDlg::CompleteConstruction()
     master_layout->Add(m_sliders_ok_cancel_layout, 0, 1, 3, 1);
     SetLayout(master_layout);
 
+    using std::placeholders::_1;
```

Comment:
You can also use

`using namespace std::placeholder;`

to make all placeholders visible in one statement.

Reply:
Not in general; it often results in ambiguity with the `boost::placeholders::_1`